src/qibosoq/__main__.py

```diff
@@ -27,9 +27,26 @@ def define_logger():
     return new_logger
 
 
-logger = define_logger()
+def define_program_logger():
+    """Define qick logger format and handler"""
 
-HOST = "192.168.0.72"  # Server address
+    new_logger = logging.getLogger("qick_program")
+    new_logger.setLevel(logging.DEBUG)
+
+    filename = "/home/xilinx/logs/program.log"
+    formatter = logging.Formatter("%(levelname)s :: %(asctime)s\n%(message)s", "%Y-%m-%d %H:%M:%S")
+
+    handler = logging.handlers.RotatingFileHandler(filename, mode="w", backupCount=3)
+    handler.setFormatter(formatter)
+
+    new_logger.addHandler(handler)
```

Comment:
Consider splitting this into a separate function, because it is just a function of `filename`, and it is almost the exact same of the one above.